wallet-sdk/src/main/kotlin/org/stellar/walletsdk/Data.kt

```diff
@@ -165,8 +177,40 @@ data class SignerWeight(
   val recoveryServer: Int,
 )
 
+enum class StellarOperationTypes(val type: String) {
+  CREATE_ACCOUNT("create_account"),
+  PAYMENT("payment"),
+  PATH_PAYMENT_STRICT_RECEIVE("path_payment_strict_receive"),
+  PATH_PAYMENT_STRICT_SEND("path_payment_strict_send"),
+  ACCOUNT_MERGE("account_merge"),
+  LIQUIDITY_POOL_DEPOSIT("liquidity_pool_deposit"),
+  LIQUIDITY_POOL_WITHDRAW("liquidity_pool_withdraw"),
+  CLAIM_CLAIMABLE_BALANCE("claim_claimable_balance")
+}
+
 enum class StellarTomlFields(val text: String) {
   SIGNING_KEY("SIGNING_KEY"),
   TRANSFER_SERVER_SEP0024("TRANSFER_SERVER_SEP0024"),
   WEB_AUTH_ENDPOINT("WEB_AUTH_ENDPOINT")
 }
+
+data class WalletAsset(val id: String, val code: String, val issuer: String)
+
+data class WalletOperation(
+  val id: Long,
+  val date: String,
+  val amount: String,
+  val account: String,
+  val asset: List<WalletAsset>,
+  val type: WalletOperationTypes,
+  val rawOperation: OperationResponse,
+)
+
+enum class WalletOperationTypes() {
+  SEND,
+  RECEIVE,
+  DEPOSIT,
+  WITHDRAW,
+  SWAP,
+  OTHER
+}
```

Comment:
Will we add some per-wallet-operation functionality in the future?
I think most likely yes, in that case maybe something like 
```suggestion
sealed interface WalletOperation {
  val id: Long
  val date: String
  val amount: String
  val account: String
  val asset: List<WalletAsset>
  val rawOperation: OperationResponse
}

data class SendWalletOperation(
  override val id: Long,
  override val date: String,
  override val amount: String,
  override val account: String,
  override val asset: List<WalletAsset>,
  override val rawOperation: OperationResponse,
) : WalletOperation
// Other operations
```
would be better

Reply:
At least initially, we'll be opinionated about what the formatted operation is. We can make it more generic later if it's needed.